package/src/components/MessageList/InlineUnreadIndicator.tsx

```diff
@@ -1,47 +1,38 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import Svg, { Defs, LinearGradient, Rect, Stop } from 'react-native-svg';
 
 import { useTheme } from '../../contexts/themeContext/ThemeContext';
 import { useTranslationContext } from '../../contexts/translationContext/TranslationContext';
-import { useViewport } from '../../hooks/useViewport';
-
-const styles = StyleSheet.create({
-  container: {
-    alignItems: 'center',
-    justifyContent: 'center',
-    padding: 10,
-    width: '100%',
-  },
-  text: {
-    fontSize: 12,
-  },
-});
 
 export const InlineUnreadIndicator = () => {
   const {
     theme: {
-      colors: { bg_gradient_end, bg_gradient_start, grey },
+      colors: { grey, light_gray },
```

Comment:
This seems like a breaking change, is it intentional ?

Reply:
It is to align the design and get rid of unnecessary SVGs.

Reply:
I mean that's fine but it's a breaking change in existing components, integrators might be getting issues (just like we did for the sender/receiver message background colors)

Reply:
These should be fine, as it doesn't depend on any props change or dep change as such since now the design is much simpler. If you disagree, I can revert, but I think the new changes are for the better. And integrators can easily customize the component on their own in whatever way they would like.

Reply:
They can do that, yes - all I'm saying is that if they've overriden `bg_gradient_end` and `bg_gradient_start` for example to fit their needs exactly, it doesn't mean that `grey` and `light_gray` will still fit their needs (even if they override these). There are also things that they might no longer be able to achieve since `grey` and `light_gray` affect other spots in the SDK too.

We change the contract on what theming affects specifically, hence why I think it's breaking - even if for the better.

Reply:
I don't mind either way if you think it's okay but we may get support tickets for this, and then it's going to be hard to fix because it'll be breaking for everyone else relying on the new values :D

Reply:
Nah lets keep this and not use SVG and gradient colors in the SDK.

Reply:
I'm against doing this as a breaking change, especially since it doesn't matter if we stop using them now since integrators could be relying on these - but as you see fit. Resolving this